src/modal/modal.scss

```diff
@@ -2,154 +2,194 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 @import '../icon/index';
+@import '../surface/index';
 
-@mixin iui-modal {
+@mixin iui-modal-backdrop {
   @include iui-reset;
   z-index: 999;
+  isolation: isolate;
   position: fixed;
-  top: 0;
-  left: 0;
+  inset: 0;
   width: 100%;
   height: 100%;
   @include themed {
     background-color: rgba(0, 0, 0, t(iui-opacity-4));
   }
 
-  // #region Small Modal Animations for CSS
+  // Small/Resizable Modal Animations for CSS
   visibility: hidden;
   opacity: 0;
-  @media (prefers-reduced-motion: no-preference) {
-    transition: visibility $iui-speed-instant linear $iui-speed-fast, opacity $iui-speed-fast ease-out;
-  }
+  transition: visibility $iui-speed-instant linear $iui-speed-fast, opacity $iui-speed-fast ease-out;
 
   &.iui-modal-visible {
     visibility: visible;
     opacity: 1;
-    // remove delay for entry animation so it's instantly visible
+    // remove delay for entry animation so modal is instantly visible
     transition-delay: $iui-speed-instant;
   }
-  // #endregion Small Modal Animations for CSS
+}
 
-  > .iui-modal-dialog {
-    position: absolute;
+@mixin iui-modal {
+  @include iui-surface($border-radius: $iui-border-radius, $elevation: $iui-elevation-24);
```

Comment:
Need to make Surface IE friendly:
![image](https://user-images.githubusercontent.com/36186912/165909102-512e7f3c-9d93-477f-af2a-1070dbc89ca0.png)

Reply:
Reverted